bindings/wasm/examples/public/examples.js

```diff
@@ -20,8 +20,11 @@ export const examples = {
       // see the static API - these functions can also be used bare. Use
       // console.log() to print output (lower-right). This editor defines Z as
       // up and units of mm.
-
-      const box = cube([100, 100, 100], true);
+      const {sphere} = Manifold;
+      const {square} = CrossSection;
+      const box = square([70, 70], true)
+                      .offset(15, 'Round')
+                      .extrude(100, 0, 0, [1, 1], true);
```

Comment:
I appreciate the idea, but I'm intentionally keeping this first landing-page example super simple. Let's revert this for now. We should add a cool example that showcases more of the CrossSection API though at some point.